src/lib/dependencies.ts

```diff
@@ -65,9 +75,14 @@ export function resolveAllDependencies (options: Options): Promise<DependencyTre
  */
 export function resolveDependency (dependency: Dependency, options: Options): Promise<DependencyTree> {
   const { type, location, raw, meta } = dependency
-
   if (type === 'registry') {
+    // DEBUGGING
+    console.log('resolveDependency starts', location, raw)
```

Comment:
Sure. It must be removed after the test issue is resolved. Do you have any gut feeling hint about that? Could it be throttling issue on the server?